Makefile

```
@@ -1,3 +1,11 @@
+BUILD      ?= build
+USE_BUILDX ?=
+
+# Setup buildx flags
+ifneq ("$(USE_BUILDX)","")
+BUILD = buildx build --platform=linux/arm64 -o type=docker
```

Do we need to fix it with `--platform=linux/arm64`? I suppose if we provide no `--platform`, the default value is decided based on your machine and `TARGETARCH` is set to arm64.

```suggestion
BUILD = buildx build -o type=docker
```

---

This is for `amd64` guys.

---

Got it, so what you want to do is building the image on amd64 and run it arm64.